jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/resources/i18n/Constants.properties

```diff
@@ -179,4 +179,5 @@ TimerInstanceRescheduled=Timer instance {0} rescheduled
 FromNow=From now
 FromActivationTime=From activation time
 ShowInstanceBadges=Show instance badges
-TimerDelayHelp=In seconds
\ No newline at end of file
+TimerDelayHelp=In seconds
+Container_Id= Container Id
```

Comment:
remove extra space

Reply:
ok

Reply:
I think the name at filters should point to 'Deployment'  instead 'Container Id' regarding that details are not displayed directly on any table column, but you can find them at Process Instance Details tab as 'Deployment'.

Reply:
I think you are right, it need align Process Instance Details